apps/src/blockly/addons/cdoRendererThrasos.js

```diff
@@ -20,3 +22,8 @@ export default class CdoRendererThrasos extends GoogleBlockly.thrasos.Renderer {
     return new CdoConstantsProvider();
   };
 }
+
+export const CdoRendererThrasos = addInlineRowSeparators(
```

Comment:
Will we need to apply this to other renderers as well?

Reply:
In theory, we could, but we're not actually using any other renderers, so I wasn't sure if it was worth doing. We briefly entertained using the Zelos renderer for Music Lab (which doesn't have these blocks). The product input seemed to be that Zelos was too big of a change and that Thrasos was a good step towards modernizing while still be recognizable. Doing this work actually made me wonder if we should delete Zelos entirely (it's currently selectable via experiment flag). I think a few internal folks are still curious about it however so maybe it's not harmful to keep it around for internal testing.